pyface/color_dialog.py

```diff
@@ -0,0 +1,43 @@
+# (C) Copyright 2005-2020 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+""" The implementation of a dialog that allows the user to open/save files etc.
+"""
+
+from .constant import OK
+from .toolkit import toolkit_object
+
+
+ColorDialog = toolkit_object("color_dialog:ColorDialog")
+
+
+def get_color(parent, color, show_alpha=False):
```

Comment:
Similarly to `get_font` - should `parent` and `show_alpha` be passed to the actual dialog? And there are no tests for this function